### ISSUE TITLE
"zeroUnits": true warns against the 0% keyframe selector

### ISSUE THREAD

**Issue by awayken** [rossPatton/stylint#95]
When defining the keyframes for a CSS animation and using the "zeroUnits" option, Styling reports a false positive at the 0% keyframe selector. In this case, '0%' is a special selector for specifying keyframes in animations. It should be ignored from the zeroUnits rule.

keyframes.styl
```stylus
@keyframes rotate {
    0% {
        transform: rotate(0deg);
    }
    100% {
        transform: rotate(360deg);
    }
}
body {
    animation: rotate 10s infinite;
}
```

keyframes.json
```json
{
    "borderNone": true,
    "brackets": false,
    "colons": false,
    "colors": false,
    "commaSpace": true,
    "commentSpace": false,
    "cssLiteral": false,
    "depthLimit": false,
    "duplicates": true,
    "efficient": true,
    "emoji": false,
    "enforceBlockStyle": false,
    "enforceVarStyle": false,
    "extendPref": false,
    "globalDupe": false,
    "indentSpaces": 4,
    "leadingZero": true,
    "maxWarnings": 10,
    "maxWarningsKill": false,
    "mixed": false,
    "namingConvention": false,
    "namingConventionStrict": false,
    "parenSpace": false,
    "placeholders": true,
    "quotePref": false,
    "semicolons": false,
    "sortOrder": false,
    "stackedProperties": true,
    "universal": true,
    "valid": false,
    "whitespace": true,
    "zeroUnits": true,
    "zIndexDuplicates": false,
    "zIndexNormalize": false
}
```

Output:
```bash
Warning:  0 is preferred. Unit value is unnecessary
File: ..\temp-stylus\keyframes.styl
Line: 2: 0% {
```

I am using 26e58b76464e297d2a2c81f16e0d2823ca1e1777 on Windows 8.1.

**Comment by rossPatton** [comment]
this appears to be a regression. will need to make sure i add more tests to cover this case. thanks for pointing it out.

**Comment by jussi-kalliokoski** [comment]
:+1: also got bit by this

**Comment by rossPatton** [comment]
k, i will fix this soon then

**Comment by rossPatton** [comment]
@awayken @jussi-kalliokoski if you guys want to pull develop and test this, it should be fixed

once that's confirmed i'll push it up to npm

**Comment by jussi-kalliokoski** [comment]
@rossPatton confirmed fixed - I tested with `gulp-stylint`.

**Comment by awayken** [comment]
I can confirm that this passes with eee056f5bf304124b244eded482b07ccca2455ed.

**Comment by rossPatton** [comment]
:+1: 

Will try to get it up soon, probably tomorrow morning

**Comment by rossPatton** [comment]
it's up! closing
